### PR TITLE
Bugsie:This is why we can't have nice things

### PR DESCRIPTION
**Pull request by Aurorablade**
YOU PEOPLE

:cl: Fethas
bugfix: You can no longer put anything THAT IS ALREADY A CULTIST IN A SOULSTONE....AT ALL.
bugfix: Readds missing beserker robe sprite.
/:cl:

### CODE REVIEW

**Comment by Alexshreds** [comment]
soul's*

File: code/game/gamemodes/wizard/soulstone.dm [hunk 1]
```diff
@@ -26,6 +26,10 @@
 			to_chat(user, "<span class='warning'>A mysterious force prevents you from trapping this being's soul.</span>")
 			return ..()
 
+		if(iscultist(M))
+			to_chat(user, "<span class='cultlarge'>This souls is already MINE.</span>")
```
Comment (Alexshreds) [inline code/game/gamemodes/wizard/soulstone.dm]: soul*

Exaggerated ending without exclamation point...

**Comment by Fox-McCloud** [comment]
this isn't the best of check, to be honest; you can still use this on other cutlists. Doing what TG did is better:

```
	if(iscultist(M))
		to_chat(user, "<span class='cultlarge'>\"Come now, do not capture your fellow's soul.\"</span>")
		return
``

**Comment by tigercat2000** [comment]
@Fox-McCloud I specifically asked her to change that, as that was what the PR originally did. It removes a valid strategy for bringing dead cultists back into the round, because the rune of reviving is a considerable pain in the ass.

**Comment by Fox-McCloud** [comment]
I'm pretty sure that's the whole point; is that there's legitimate risk to being killed and requires resource investment on the other cultists part to get you back into play besides the "just swarm them and I'll bring you back as a juggernaught so you can keep doing the same thing".

**Comment by Aurorablade** [comment]
@Fox-McCloud what do you want me to do, revert it to my first commit before tigers suggestion?

**Comment by Crazylemon64** [comment]
@Aurorablade Merge Conflicts

**Comment by Fox-McCloud** [comment]
@Aurorablade 

You put up a strawpoll on this, twice, and both times the "no cultists in soul stones, ever" won out; I'd go ahead and go through with that.

**Comment by Aurorablade** [comment]
i will probaly do this.

File: code/game/gamemodes/wizard/soulstone.dm [hunk 2]
```diff
@@ -53,14 +53,20 @@
...
-		
+
 	if(M.has_brain_worms()) //Borer stuff - RR
 		to_chat(user, "<span class='warning'>This being is corrupted by an alien intelligence and cannot be soul trapped.</span>")
 		return ..()
 
 	if(jobban_isbanned(M, "cultist") || jobban_isbanned(M, "Syndicate"))
 		to_chat(user, "<span class='warning'>A mysterious force prevents you from trapping this being's soul.</span>")
 		return ..()
+		if(iscultist(M))
```
Comment (tigercat2000) [inline code/game/gamemodes/wizard/soulstone.dm]: this is tabbed behind jobban_isbanned and is under a return ..()

**Comment by Aurorablade** [comment]
ACURRSED MERGE CONFLICT FIXING WITH TORTISE GIT...THANK YOUR TIGERMROWLMROWLMROWLMROWL

**Comment by Aurorablade** [comment]
i am adding one more thing and will update OP..but i lost the berserker armor sprite..somehow..

File: code/game/gamemodes/wizard/soulstone.dm [hunk 3]
```diff
@@ -62,23 +62,30 @@
 		to_chat(user, "<span class='warning'>A mysterious force prevents you from trapping this being's soul.</span>")
 		return ..()
 
+	if(iscultist(M))
```
Comment (Crazylemon64) [inline code/game/gamemodes/wizard/soulstone.dm]: feh

**Pull request merged**